src/raftstore/store/peer_storage.rs

```diff
@@ -637,6 +747,7 @@ impl PeerStorage {
                                      raft_count += 1;
                                      Ok(true)
                                  }));
+        self.cache = VecDeque::new();
```

Comment:
Not necessary. If the region is inactive, preallocation will be a waste. We usually have many regions which is a big cost.